apps/web/modules/core/rate-limit/rate-limit.test.ts

```
@@ -0,0 +1,344 @@
+        arguments: ["5", expect.any(String)],
+      })
+    );
+  });
+
+  test("should calculate correct TTL for window expiration", async () => {
+    mockEval.mockResolvedValue([1, 1]);
+
+    await checkRateLimit(testConfig, "test-user");
+
+    // TTL should be between 0 and 300 seconds (window interval)
+    const ttlUsed = parseInt(mockEval.mock.calls[0][1].arguments[1]);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)